### PR TITLE
Add Aurelia

### PR DESCRIPTION
**Pull request by behzad888**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -125,6 +125,7 @@ Check out my [blog](https://blog.sindresorhus.com) and follow me on [Twitter](ht
 	- [Critical-Path (Above-the-fold) Tools](https://github.com/addyosmani/critical-path-css-tools)
 	- [Scalability](https://github.com/davidtheclark/scalable-css-reading-list)
 	- [Must-Watch Talks](https://github.com/AllThingsSmitty/must-watch-css)
+- [Aurelia](https://github.com/behzad888/awesome)
```
Comment (sindresorhus) [inline readme.md]: The repo name should be something more descriptive, like `awesome-aurelia`

**Comment by sindresorhus** [comment]
Thanks for the suggestion. Would you mind reading the [contributing guidelines](https://github.com/sindresorhus/awesome/blob/master/contributing.md) and the [awesome manifest](https://github.com/sindresorhus/awesome/blob/master/awesome.md) :)

> Your contributions are always welcome!

This is not a good contribution guideline. See example of a good one: https://github.com/sindresorhus/awesome-npm/blob/master/contributing.md

File: readme.md (hunk 1 above)
Comment (behzad888) [inline readme.md]: got it

**Comment by sindresorhus** [comment]
Adding the contribution guideline directly in the readme is noisy. Put it in a contributing.md file and rather link to it, like done here: https://github.com/sindresorhus/awesome-npm

**Comment by sindresorhus** [comment]
Added.

**Pull request closed without merging**